app/assets/javascripts/modules/AssignedToFilter.js

```diff
@@ -0,0 +1,38 @@
+moj.Modules.AssignedToFilter = {
+
+    init: function () {
+
+        if ($('#search_query_business_unit_name_filter').length === 0 ){
```

Comment:
Why do we need this as on line 10 we only operate on `#search_query_business_unit_name_filter`?